<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hub</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img_carrinho.jpg</normaloff>img_carrinho.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>981</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="tudo">
     <item>
      <layout class="QHBoxLayout" name="topbar">
       <item>
        <layout class="QVBoxLayout" name="busca_layout">
         <item>
          <widget class="QLabel" name="busca">
           <property name="text">
            <string>Busca:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="busca_edit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="advanced">
           <item>
            <layout class="QVBoxLayout" name="min_preco_layout">
             <item>
              <widget class="QLabel" name="min_preco">
               <property name="text">
                <string>Preço Min.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="min_preco_spin">
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="max_preco_layout">
             <item>
              <widget class="QLabel" name="max_preco">
               <property name="text">
                <string>Preço Max.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="max_preco_spin">
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1000000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="tipo_layout">
             <item>
              <widget class="QLabel" name="tipo">
               <property name="text">
                <string>Tipo de Produto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="tipo_combo">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>Aluguel</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Venda</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Serviço</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Todos</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="popularidade_min_layout">
             <item>
              <widget class="QLabel" name="popularidade_min">
               <property name="text">
                <string>Popularidade Min.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="popuaridade_min_spin">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="popularidade_max_layout">
             <item>
              <widget class="QLabel" name="popularidade_max">
               <property name="text">
                <string>Popularidade Max.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="popuaridade_max_spin">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="username">
           <property name="text">
            <string>Faça Login</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="login">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="registrar">
             <property name="text">
              <string>Registrar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logout">
             <property name="text">
              <string>Logout</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="error_message">
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="resultados"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
